modules/mono/glue/cs_files/Vector2.cs

```diff
@@ -116,14 +114,14 @@ public Vector2 Clamped(real_t length)
         public Vector2 CubicInterpolate(Vector2 b, Vector2 preA, Vector2 postB, real_t t)
         {
             Vector2 p0 = preA;
-            Vector2 p1 = this;
+	        var p1 = this;
             Vector2 p2 = b;
             Vector2 p3 = postB;
```

Comment:
The type declarations here should be in line with the other lines right next to it. Sure, in the `Vector2` class, you know that `var p1 = this;` has to be a `Vector2`, but simply the fact that it's different from `p0`, `p2`, `p3` reduces readability. Either make it `Vector2 p1`, or it would be fine if you declared `p0`, `p2`, and `p3` with `var` (as their types are all listed right above in the method args).

Other than this (and the one in `Vector3`), I agree with every use of `var` in this PR as of right now. Good job! :tada:

Reply:
Changed it, and squashed all relevant commits. Thanks!